### ISSUE TITLE
Blank Page Displays on Browsers that don't support Audio Worklet

### ISSUE THREAD

**Issue by AnthumChris** [GoogleChromeLabs/web-audio-samples#195]
Safari v13.0.4 shows a blank page when loading https://googlechromelabs.github.io/web-audio-samples/audio-worklet/

Console error:
```
[Error] ReferenceError: Can't find variable: OfflineAudioContext	
  _detectAudioWorklet (Components.js:210)	
  WorkletIndicator (Components.js:202)	
  build (Components.js:35)	
  Module Code (audio-worklet:52)	
  evaluate	
  moduleEvaluation
```

**Comment by AnthumChris** [comment]
@hoch Renamed to remove the Safari specifity.  This error will result on any browser that does not have `OfflineAudioContext` in global window scope.

**Comment by hoch** [comment]
Safari has OfflineAudioContext. It is prefixed with `webkit`.

It's better to cause TypeError, and then catch to display an error message.

**Comment by AnthumChris** [comment]
Ok.  The page reads "If the status indicator on the top-right corner of this page is not green, it means your browser does not support Audio Worklet. If you are using Chrome, update it to the latest version."

Currently, a blank page would show for users in browsers that produce that error.

**Comment by hoch** [comment]
@AnthumChris Does this still reproduce? I believe now all the browsers support Audio Worklet.

**Comment by AnthumChris** [comment]
@hoch I'm always an advocate for showing messages when browsers don't support things, but feel free to close this.  Or perhaps it could be nice to show users a message if they're using an older browser.

**Comment by AnthumChris** [comment]
https://github.com/GoogleChromeLabs/web-audio-samples/pull/196 fixes this `undefined` object error. Can we please merge or suggest alternatives to fix?

**Comment by hoch** [comment]
Yup. Sorry for holding up. Let's merge it now and iterate later.

**Comment by hoch** [comment]
The new PR is up. I'll merge it after some testing.
